### PR TITLE
Add fdpf, boto3, netsuitesdk, notifiers, pynput-robocorp-fork, robotf…

### PR DESCRIPTION
**Pull request by xylix**
…ramework-sapguilibrary, robotframework-seleniumtestability

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)

- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team. 

Will need to check the license stuff, but posting this here to let the CI run

### CODE REVIEW

**Comment by xylix** [comment]
@conda-forge-admin please rerender

**Comment by xylix** [comment]
@conda-forge/staged-recipes Ready for review.

File: recipes/fpdf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+test:
+  imports:
+    - fpdf
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://code.google.com/p/pyfpdf
```
Comment (wolfv) [inline recipes/fpdf/meta.yaml:34]: updated URL points to github.

File: recipes/fpdf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  summary: Simple PDF generation for Python
+  license: LGPLLR
```
Comment (wolfv) [inline recipes/fpdf/meta.yaml:36]: This file tells me it's regular LGPL v3? https://github.com/reingart/pyfpdf/blob/master/LICENSE

File: recipes/pynput-robocorp-fork/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - enum34  # [py==27]
+    # pyobjc-framework versions need to match
+    - pyobjc-framework-quartz ==5.3  # [osx]
+    - pyobjc-framework-cocoa ==5.3  # [osx]
+    - python
+    - python-xlib >=0.17
+    - six
+
+test:
+  # imports:
```
Comment (wolfv) [inline recipes/pynput-robocorp-fork/meta.yaml:34]: Does it not work?

File: recipes/robotframework-sapguilibrary/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - pywin32 >=222
+    - robotframework >=2.9
```
Comment (wolfv) [inline recipes/robotframework-sapguilibrary/meta.yaml:25]: Is there some reasonable upper bound like <3?

File: recipes/rpaframework-recognition/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - numpy ==1.18.4
```
Comment (wolfv) [inline recipes/rpaframework-recognition/meta.yaml:24]: is it really necessary to pin numpy so strongly?
In general, numpy is pinned by the conda-forge-pinnings which ensure consistency across the ecosystem. Whenever a pin is updated in the conda-forge pinnings, a migration is triggered. 

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml

If you pin numpy this strongly, installing this package with other packages depending on numpy (scipy, scikit-learn ... ) might not be possible.

File: recipes/fpdf/meta.yaml (hunk 2 above)
Comment (xylix) [reply recipes/fpdf/meta.yaml:36]: Hm apparently. Fixing.

File: recipes/pynput-robocorp-fork/meta.yaml (hunk 3 above)
Comment (xylix) [reply recipes/pynput-robocorp-fork/meta.yaml:34]: Pynput will not import without X11 / other graphical env. And didn't seem worth the hassle to start working yum configurations. Didn't find a way to only skip the test on linux

File: recipes/robotframework-sapguilibrary/meta.yaml (hunk 4 above)
Comment (xylix) [reply recipes/robotframework-sapguilibrary/meta.yaml:25]: For python? Or RF? For robot framework there is no reasonable upper bound.

File: recipes/rpaframework-recognition/meta.yaml (hunk 5 above)
Comment (xylix) [reply recipes/rpaframework-recognition/meta.yaml:24]: Will try migrating to an ecosystem pin.

Comment (xylix) [reply recipes/rpaframework-recognition/meta.yaml]: Fixed in e15a50ceb167bc7c4466fecfe215381ea6972524

File: recipes/robotframework-sapguilibrary/meta.yaml (hunk 4 above)
Comment (wolfv) [reply recipes/robotframework-sapguilibrary/meta.yaml:25]: ok.

File: recipes/pynput-robocorp-fork/meta.yaml (hunk 3 above)
Comment (wolfv) [reply recipes/pynput-robocorp-fork/meta.yaml:34]: you can skip the tests by adding `#  [not linux]` at the end of the line. But it's fine.

File: recipes/pynput-robocorp-fork/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    - sphinx >=1.3.1
+  run:
+    - enum34  # [py==27]
+    # pyobjc-framework versions need to match
+    - pyobjc-framework-quartz ==5.3  # [osx]
+    - pyobjc-framework-cocoa ==5.3  # [osx]
+    - python
+    - python-xlib >=0.17
+    - six
+
```
[the remark this answers is not in the document]
Comment (xylix) [reply recipes/pynput-robocorp-fork/meta.yaml:32]: The changes we did is basically to remove `tty` support from `pynput` to make it behave more nicely in graphical linux environments without linux-evdev headers installed. This means that they are not really changes that upstream would like.

Number 2. is possible, and makes sense for our  `pypi` package as well. I will pull `pynput` out of this PR and add this differently importable pynput separately.

File: recipes/pynput-robocorp-fork/meta.yaml (hunk 3 above)
Comment (xylix) [reply recipes/pynput-robocorp-fork/meta.yaml]: Okay good to know!

**Pull request merged**